templates/file.j2

```diff
@@ -39,7 +39,14 @@
 {% if data.state == PortState.PORTED%}
 <div class="alert alert-success">This file has been ported!</div>
 {% elif data.state == PortState.IN_PROGRESS %}
-<div class="alert alert-warning">This file is currently being ported at <a href="https://github.com/leanprover-community/mathlib4/pull/{{data.status.mathlib4_pr}}">#{{data.status.mathlib4_pr}}</a>.</div>
+<div class="alert alert-warning">This file is currently being ported at <a href="https://github.com/leanprover-community/mathlib4/pull/{{data.status.mathlib4_pr}}">#{{data.status.mathlib4_pr}}</a>.
+{%- for label in data.labels %}
+    <a href='https://github.com/leanprover-community/mathlib4/pulls?q=is%3Apr+is%3Aopen+label%3A"{{label.name}}"'>
+        <span class="badge rounded-pill" style="background-color:#{{label.color}};color:{{label.text_color}}">
+        {{label.name}}</span>
+    </a>
+{%- endfor %}
```

Comment:
You can merge the `a` and `span` tag into one; see the latest `index` page.

Reply:
Yes, that's nicer, it makes the weird blue line between the pills go away...